py/specter/util/util.py

```diff
@@ -39,9 +31,19 @@ def __init__(self, x, y, data):
         data[ix, iy, ...] : 3 or more dimensional array of data to interpolate
             first two coordinates are x and y
         """
-        self.x = np.array(x)
-        self.y = np.array(y)
-        self.data = np.array(data)
+        self.x = np.array(x).astype(np.float64)
+        self.y = np.array(y).astype(np.float64)
+        self.data = np.array(data).astype(np.float64)
+
+    #new method
+    @jit(nopython=True)
+    def new_interp(dx,dy,ix,iy,datacopy):
```

Comment:
See comments in `SpotGridPSF` about using `numba.jit` (stylistic clarity), moving the JIT-ed function outside of the class, and documenting the inputs/outputs.  Same comments apply here.

Reply:
Will apply the same stylistic changes later, for now reverted to master branch version.